assets/src/edit-story/components/form/numeric.js

```diff
@@ -99,38 +99,84 @@ function Numeric({
 }) {
   const isMultiple = value === MULTIPLE_VALUE;
   const placeholder = isMultiple ? __('multiple', 'web-stories') : '';
-  const [dot, setDot] = useState(false);
   const [inputValue, setInputValue] = useState(value);
   const ref = useRef();
+  const targetFormRef = useRef(null);
 
-  useEffect(() => setInputValue(value), [value]);
+  const isIterimValue = useCallback(
+    (val) =>
+      // Allow a single "-" if negative values are allowed
+      (canBeNegative && val === '-') ||
+      // Allow string of 0s
+      /^[0]+$/.test(val) ||
+      // Allow starting and ending with decimal point if float
+      (float && (val.startsWith(DECIMAL_POINT) || val.endsWith(DECIMAL_POINT))),
+    [float, canBeNegative]
```

Comment:
I think here is where we'd want to allow an empty value as `interim` so we can fully backspace out the input.

Reply:
Good catch!  I think I have a way of getting this working again without introducing the other bugs the new pattern fixes 🤔

Reply:
Okay, I wanted to keep all the logic of `inputValue` and `value` clean, since I do believe it's working for all the other edge cases.  And I realized I could capture the specific state of a user deleting all the content by simply checking if `newValue`  (within the `onChange` handler) is empty or not.  If it's empty then we're in the "Empty" state, which _only_ changes display logic (not `inputValue` to `value` logic), and as soon as `newValue` has any value then the state becomes `Active` and everything works as expected... 

Here's a gif of all the delete and re-input action:
![full_delete](https://user-images.githubusercontent.com/40646372/93840748-55771800-fc46-11ea-8a5c-3426eb4c2eb2.gif)